tests/dns/dns-rcode/test.rules

```diff
@@ -0,0 +1,5 @@
+# Should alert in client direction.
+alert dns any any -> any any (dns.rcode:3; sid:1; rev:1;)
```

Comment:
With the hex edit of pcap, do you think you can craft a pcap with rcode 23, for us to test? :)